lilypad/server/_utils/auth.py

```diff
@@ -0,0 +1,105 @@
+import json
+import os
+from typing import Annotated
+from uuid import UUID
+
+from fastapi import Depends, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer
+from jose import JWTError, jwt
+from pydantic import ValidationError
+from sqlmodel import Session, select
+
+from ..db import get_session
+from ..models import (
+    OrganizationPublic,
+    OrganizationTable,
+    UserOrganizationTable,
+    UserPublic,
+    UserRole,
+    UserTable,
+)
+from ..settings import Settings, get_settings
+
+LOCAL_TOKEN = "local-dev-token"
+
+
+def create_jwt_token(
+    user_data: UserPublic,
+) -> str:
+    settings = get_settings()
+    return jwt.encode(
+        json.loads(user_data.model_dump_json()),
+        settings.jwt_secret,
+        algorithm=settings.jwt_algorithm,
+    )
+
+
+if os.getenv("LILYPAD_LOCAL", "false").lower() == "true":
+
+    async def oauth2_scheme(token: str | None = None) -> str:  # pyright: ignore[reportRedeclaration]
```

Comment:
nit: can we avoid the type ignore if we name this something like `get_local_token` and set `oauth2_scheme = get_local_token`?

Reply:
i tried what you did and still got an error, so not sure what the best way to handle this is.

Reply:
ok just leave it as-is for now